elections/2023-TOC/README.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "Knative TOC Election - 2023 Voters Guide"
+linkTitle: "2023 Voters Guide"
+weight: 30
+type: "docs"
+---
+
+# 2023 Voters Guide - Knative TOC Election
+
+## Purpose
+
+The role of this election is to fill the one (1) seat due for election this year on the [Knative Technical Oversight Committee]. The elected
+member will serve a two (2) year term.
+
+## Background
+
+This election will shape the future of Knative as a community and project.
+While WGs help shape the individual direction of the project, the
+[TOC Charter] covers the technical direction, health of the project
+and community as a whole.
+
+## Eligibility
+
+Please refer to the [TOC Election Charter] for:
+
+- [Eligibility for candidacy]
+- [Eligibility for voting] 
+
+The list of eligible voters is stored [in the community repository](https://github.com/knative/community/elections/2023-TOC/), but it is easier
+to check if you are eligible by logging into [Elekto]. See Voting Process below for more details.
+
+### Schedule
+
+| Date         | Event                    |
+| ------------ | ------------------------ |
+| May 2        | Announcement of Election |
+| May 10       | All candidate nominations are due by 23:59 UTC (4:59pm Pacific) |
+| May 13       | Election Begins via Elekto UI |
+| May 24       | Voting exception forms are due by 23:59 UTC (4:59pm Pacific)
+| May 27       | Election Closes by 23:59 UTC (4:59pm Pacific) |
+| June 1       | Announcement of Results |
+
+## Election Officers
+
+In an effort to run a fair and transparent election, the following people
+have been asked by the Steering Committee to run this election:
+
+- [Dawn Foster](https://github.com/geekygirldawn)
+- [Ali Ok](https://github.com/aliok)
+
+You can reach us by emailing elections@knative.team
```

Comment:
Who receives mails to this address? Are we sure that this inbox is working?

Reply:
Ah, good question! That alias is configured in groups.yaml and I've updated it there as another commit within this PR. We'll need to wait until the PR is merged and then test it to make sure it's working.